### DISCUSSION TITLE
使用Ubuntu22.04时，无法加载封面图

### DISCUSSION THREAD

**Discussion by lanyeeee** [lanyeeee/jmcomic-downloader#31] [General]
原因：Ubuntu22.04的`libwebkit2gtk-4.1`库有BUG，`Tauri v2`依赖这个库，而本项目依赖于`Tauri v2`
解决办法：没啥好办法，只能等上游修复

**Comment by taotieren** [comment]
将这个下载器打包到 Arch Linux 那边的 AUR 仓库了，运行测试也是显示白屏。。。。

**Reply by lanyeeee** [reply]
看这个  https://github.com/lanyeeee/jmcomic-downloader/discussions/32
